src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -24,79 +24,76 @@
    {:db (assoc-in db [:wallet :ui :send :select-address-tab] tab)}))
 
 (rf/reg-event-fx :wallet/suggested-routes-success
- (fn [{:keys [db]} [suggested-routes timestamp]]
-   (when (= (get-in db [:wallet :ui :send :suggested-routes-call-timestamp]) timestamp)
```

Comment:
Check was removed because the call to the endpoint and where it is successfully (signals) handled has changed

Reply:
The asynchronous endpoint can take would override the current call if called again while the request is pending. 

So, I don't think this timestamp check is useful in this context

Reply:
With this asynchronous code. Should we still debounce the calls, since the async code on status go uses a queue to resolve this and current running requests are overridden till there is a result. 

cc: @J-Son89

Reply:
@briansztamfater, @vkjr probably know better.

Reply:
> The asynchronous endpoint would override the current call if called again while the request is pending.
> 
> So, I don't think this timestamp check is useful in this context

I think we can remove this check 👍

Reply:
How about the debounce, do you think it is safe to remove? cc: @briansztamfater @vkjr

Reply:
@BalogunofAfrica, yes, I think we should leave it. The debounce is there to prevent sending multiple events when user changes something on token input screen, for example when user types in the amount to send. We do not want to invoke routes recalculation on each change, only when user stopped for a bit)
Even if it is resolved on `status-go` side, we don't have to rely on that implementation, it can change one day without us knowing and we will get unexpected performance issue)